wicket-util/src/main/java/org/apache/wicket/util/time/Duration.java

```diff
@@ -1,555 +1,565 @@
-/*

```

Comment:
Why this class was re-formatted?

Reply:
Not following you. Class seems to have been removed.

Reply:
@reiern70 it is reformatted, please load full diff and scroll to the middle :)

Reply:
The only difference I can spot is this...

![image](https://user-images.githubusercontent.com/462655/58690662-c714be00-8392-11e9-9050-c88a730d8b82.png)

And this is a method I have added.

Reply:
The question is: "why the diff is so huge"?
I guess it might be caused by massive line ending change

Reply:
Hum... I remember my IDE complaining about that and I might have chosen the wrong option.

Reply:
![image](https://user-images.githubusercontent.com/462655/58693295-1a8a0a80-8399-11e9-89e4-e0126bd6fba5.png)

One of master has CRLF. Which one is correct?

Reply:
Duration class have 

![image](https://user-images.githubusercontent.com/462655/58697249-e830db00-83a1-11e9-9d04-4d2b5f32b1a6.png)

and  AjaxSelfUpdatingTimerBehavior

![image](https://user-images.githubusercontent.com/462655/58697306-072f6d00-83a2-11e9-80f0-f2eb76211567.png)

Reply:
I don't know which one is correct :(
But re-formatting made diff absolutely unusable

Reply:
> I don't know which one is correct :(
> But re-formatting made diff absolutely unusable

My problem was I had

 git config --global core.autocrlf true